acceptance_tests/features/steps/supporttool_ui.py

```diff
@@ -206,9 +206,9 @@ def click_email_action_rule_button(context, email_column):
     context.browser.find_by_id('selectActionRuleType').click()
     context.browser.find_by_value('Email').click()
     context.browser.find_by_id('selectActionRuleEmailPackCode').click()
-    context.browser.find_by_id(context.pack_code).click()
+    context.browser.find_by_id(context.pack_code, wait_time=20).click()
     context.browser.find_by_id('selectActionRuleEmailColumn').click()
-    context.browser.find_by_id(email_column).click()
+    context.browser.find_by_id(email_column, wait_time=20).click()
```

Comment:
This way of doing things with a long static wait will start to seriously increase the test time, are there any more dynamic alternatives? Could we perhaps try a shorter wait to begin with?

Reply:
Double checked the this:
https://splinter.readthedocs.io/en/latest/matchers.html?highlight=wait_time#matchers

Also I debugged the new bits locally, they don't wait 20 secs before completing if successful, pretty instant.  I think looking at the code the default value is 2, but this is presumably implemented as some sort of retry loop